### PR TITLE
Update Education track page

### PR DESCRIPTION
**Pull request by bennuttall**
I've added some information and @MissPhilbin and @ntoll have contributed

### CODE REVIEW

File: pages/education.md [hunk 1]
```diff
@@ -1,38 +1,71 @@
...
+
+Kids (and parents): come learn how to build the future at PyCon UK. Experts in
+computing education and professional programmers will be on hand to deliver a
+series of fun, practical and interesting workshops that'll help you learn the
+basics of programming, develop your computing skills and make cool projects.
+
+### What's on
+
+**[Picademy](https://www.raspberrypi.org/training/picademy/)**: the Raspberry Pi Foundation's free face-to-face training programme that supports educators in
+their digital making and computing journey. It's a two-day CPD event held at
```
Comment (alexwlchan) [inline pages/education.md:24]: minor: I had to google “CPD event” to know what the acronym stands for, but I’m guessing you expect teachers to understand this?

File: pages/education.md [hunk 2]
```diff
@@ -1,38 +1,71 @@
...
+PyCon UK on Thursday and Friday, and continues as educators are welcomed into a
+community of passionate Python practitioners. Picademy is free to attend, but
+places are limited. [Apply now](https://docs.google.com/forms/d/e/1FAIpQLSfSjics2BeFPfDToYsnhVxGNhMNHf4YFaDhuiqspwHHQ14QYA/viewform)!
+
+**[Code Club](https://www.codeclub.org.uk/)**: the ever popular PyConUK Code
```
Comment (alexwlchan) [inline pages/education.md]: “PyConUK” needs a space, which is what’s breaking the build

File: pages/education.md [hunk 3]
```diff
@@ -1,38 +1,71 @@
...
+[financial assistance](http://2017.pyconuk.org/financial-assistance/) to help
+cover the cost of attending the conference.
 
-Tickets for the kids' day will go on sale soon, and will cost £5.
+Children's tickets for Saturday are just £5. Parents and guardians accompanying
+children do not require a ticket. A free picnic lunch will be provided for both
+kids and parents.
 
+If you have been granted a place on the Picademy course, you will not need a
+PyConUK ticket unless you are also planning to attend on Saturday.
```
Comment (alexwlchan) [inline pages/education.md]: ditto above

**Review by alexwlchan — CHANGES_REQUESTED** [review]

File: pages/education.md (hunk 1 above)
Comment (bennuttall) [reply pages/education.md:24]: Oh I thought @ntoll had changed this but I guess he just commented. @MissPhilbin - it's your call.

**Review by alexwlchan — APPROVED** [review]
Okay, I’m happy to merge this as-is. Shout if you want to change CPD, and if not I’ll merge it when I get home.

**Comment by bennuttall** [comment]
Thanks! :+1:

**Pull request merged**